### PR TITLE
run tests using fades to install the necessary dependencies

### PR DESCRIPTION
**Pull request by ricardokirkner**
With these changes it's now possible to simply run the tests using the 'test' script without previously having to install the dependencies system-wide or in a virtualenv.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -141,6 +141,9 @@ On the other hand if you pass the ``-q`` or ``--quiet`` parameter, *fades*
 will not show anything (unless it has a real problem), so the original
 script stderr is not polluted at all.
 
+Sometimes, you want to run a script provided by one of the dependencies
+installed into the virtualenv. *fades* supports this via the ``-x`` (
+or ``--executable`` argument).
```
Comment (facundobatista) [inline README.rst]: Change for "--exec"

File: README.rst [hunk 2]
```diff
@@ -176,19 +182,7 @@ Several instructions to install fades in different systems.
...
-You can install it in Ubuntu/Debian with::
-
-    apt-get install python-xdg
-
-And on Archlinux with::
-
-    pacman -S python-xdg
-
-Fades also needs the `virtualenv <https://virtualenv.pypa.io/en/latest/>` package to
+Fades needs the `virtualenv <https://virtualenv.pypa.io/en/latest/>` package to
```
Comment (gilgamezh) [inline README.rst]: This would be a sepparated explain

File: fades/main.py [hunk 3]
```diff
@@ -82,6 +82,9 @@ def go(version, argv):
     parser.add_argument('-p', '--python', action='store',
                         help=("Specify the Python interpreter to use.\n"
                               " Default is: %s") % (sys.executable,))
+    parser.add_argument('-x', '--executable', action='store_true',
```
Comment (facundobatista) [inline fades/main.py]: Change for "exec"

Comment (gilgamezh) [inline fades/main.py]: As we talked `--exec` is a better name. :)

File: test [hunk 4]
```diff
@@ -10,9 +10,10 @@ else
...
 
+FADES='./bin/fades -r requirements.txt'
 
-nosetests3 -v -s $TARGET
-flake8 $TARGET --max-line-length=99 --select=E,W,F,C,N
+$FADES -x nosetests -- -v -s $TARGET
+$FADES -x flake8 -- $TARGET --max-line-length=99 --select=E,W,F,C,N
 
 # On Archlinux rst2html is rst2html2
 if hash rst2html > /dev/null 2>&1; then
```
Comment (facundobatista) [inline test]: This magic of "how rst2html is named in Arch" is no longer needed, as now it's used from the virtualenv

**Pull request merged**